build.gradle

```diff
@@ -24,7 +24,7 @@ allprojects {
 
     }
 
-    project.version = "1.0.0-beta11"
+    project.version = "1.0.0-beta12-SNAPSHOT"
```

Comment:
don't we need to update core sdk SNAPSHOT here as well?

Reply:
updated Core SDK version